fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -314,6 +316,11 @@ public int getTimeoutCount(Event event) {
                 result.put(prefix + "_micros", counter.timeNanos.get() / 1000);
             }
         }
+        // now add recorded timeout events to map
+        for (Event timeoutEvent : timeoutEvents) {
+            String timeoutPrefix = timeoutEvent.name().toLowerCase(Locale.ROOT);
+            result.put(timeoutPrefix + "_timeout_micros", getTimeoutTimeNanos(timeoutEvent) / 1000);
```

Comment:
Could this also log the count for each timeout (with a `"_timeout_count"` suffix, probably)?